unify-front-end/supabase/functions/request-group/index.ts

```
@@ -0,0 +1,122 @@
+// Basic email validation
+function isValidEmail(email: string): boolean {
+  // Simple but robust regex for email validation
+  const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+  return emailRegex.test(email) && email.length <= 254;
+}
+
+// Sanitize string for use in email subject (prevent header injection)
+function sanitizeForSubject(str: string): string {
+  return str
+    .replace(/[\r\n\t]/g, ' ')  // Replace CR, LF, TAB with space
+    .replace(/[\x00-\x1f\x7f]/g, '')  // Remove other control characters
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)